### ISSUE TITLE
Invalid project name, a project already exists with that name

### ISSUE THREAD

**Issue by rajamukherji** [readthedocs/readthedocs.org#7606]
## Details

* Read the Docs project URL: https://radb.readthedocs.io/en/latest/
* Build URL (if applicable):
* Read the Docs username (if applicable): rajamukherji

## Expected Result

There does not seem to be an existing project at https://radb.readthedocs.io/en/latest/. So I would like to create radb project for https://github.com/rajamukherji/radb

## Actual Result

Invalid project name, a project already exists with that name

**Comment by stsewd** [comment]
Have you tried contacting the author https://readthedocs.org/projects/radb/ https://gitlab.com/ppokhrel11?

We have a policy for abandoned projects https://docs.readthedocs.io/en/stable/abandoned-projects.html but first you need to try to contact the author.

**Comment by rajamukherji** [comment]
Thanks, I wasn't aware of how to identify the author. I will contact them and follow the abandoned projects policy if I do not get a reply.

**Comment by ssbarnea** [comment]
To be honest, I am very confused by the fact that how to find the username that owns a project slur (namespace) is impossible for us and the process is not documented.

Somehow RTD thinks that we have the power to guess not only RTD username but also his email address in order to contact him to takeover the project back. But with all this information missing we are clueless.

While I can see why email would count as private info and should not be make public, the RTD username is not really a private and probably it can help finding the user, especially as in most cases someone else might had accidentally reserved the name and forgot to add other team admins.

**Comment by humitos** [comment]
Hi @ssbarnea! I replied you this on Twitter at https://twitter.com/readthedocs/status/1614910869070282752. I also saw that you sent us a support request via email with the same question and also commented here. Please, contact us only once for the same issue/question.

**Comment by ssbarnea** [comment]
@humitos Thanks for the help but in the end that was not what I needed. Still, I was able to use a trick in RTD ui to dig for who own the project.

Basically I used `https://readthedocs.org/profiles/johndoe/` url to check user profiles for those that I was suspected as might be owning the orphan project and I found the users.

Obviously that my hack works only when you have a suspect list, which you do not always have.

The real problem is that the error message about importing a project due to a project namespace being already taken is incomplete and unhelpful, as it does not expose the (rtd) usernames of those that currently own the namespace.

Displaying them is not a security issue, username is not a private information. By not doing this RTD does create extra support strain as our only escape is to create a support ticket to get help.
